### ISSUE TITLE
Bayesian Polishing failing for some dataset

### ISSUE THREAD

**Issue by delarosatrevin** [scipion-em/scipion-em-relion#212]
**Describe the bug**
Bayesian polishing is failing with the. following issue:

```
ERROR:
00050:   There is no movie metadata STAR file for any micrographs!
00051:   WARNING: This is not a particle STAR file but contains
rlnImagePixelSize column.
00052:   Pixel size in rlnImagePixelSize will be copied to
rlnMicrographPixelSize column. Please make sure this is correct!
00053:   in:
/mnt/cephfs/projects/2019032700_Legacy_bjornf/bjornf/relion/relion/src/jaz/micrograph_handler.cpp,
line 261
00054:   ERROR:
00055:   There is no movie metadata STAR file for any micrographs!
00056:   === Backtrace  ===
```

**To Reproduce**
Just happened with a 1000 movies dataset, it does not seem to happen with the tutorial

**Comment by delarosatrevin** [comment]
There were a couple of issues here:
* the micName should be used to really map between particles and movies and metadata star file
* the local motion block should be written per movie and the whole block omitted if not coefficients (that is what happens for movies that fail to align)

**Comment by delarosatrevin** [comment]
Fixed in c5425ee97b56f2988cd139338fdc50b54fca6872
